package.json

```diff
@@ -29,7 +29,7 @@
     "homoglyph-finder": "1.1.1",
     "https-browserify": "0.0.1",
     "identicon.js": "github:status-im/identicon.js",
-    "instabug-reactnative": "git+https://github.com/status-im/instabug-reactnative.git",
```

Comment:
How come you don't update the status-im one? We should use that, not our own forks.

If it is a permission issue and you do this to be unblocked: OK - please ping Carl and/or Nabil and he will sort you (us) out.

Reply:
^ @janherich